content/en/blog/2024/v1-apis/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Introducing Istio v1 APIs "
+description: Announcing the v1 Networking, Security and Telemetry APIs. 
+publishdate: 2024-05-13
+attribution: Whitney Griffith - Microsoft
+keywords: [istio, traffic, security, telemetry, API]
+target_release: 1.22
+---
+
+Istio has several [Networking](/docs/reference/config/networking/), [Security](/docs/reference/config/security/) and [Telemetry](/docs/reference/config/telemetry/) APIs that are crucial for ensuring the secure, seamless connectivity, and effective monitoring of services within the Service Mesh layer. These APIs are used on thousands of clusters across the world, securing and enhancing critical infrastructure. As a reflection of the stability, adoption, and value of these resources, the Istio community has decided to promote many of our CRDs to `v1` in Istio 1.22.
+
+In Istio 1.22 we are happy to announce that a concerted effort has been made to graduate the below APIs to `v1`:
+* [Destination Rule](/docs/reference/config/networking/destination-rule/)
+* [Gateway](/docs/reference/config/networking/gateway/)
+* [Service Entry](/docs/reference/config/networking/service-entry/)
+* [Sidecar](/docs/reference/config/networking/sidecar/)
+* [Virtual Service](/docs/reference/config/networking/virtual-service/)
+* [Workload Entry](/docs/reference/config/networking/workload-entry/)
+* [Workload Group](/docs/reference/config/networking/workload-group/)
+* [Telemetry API](/docs/reference/config/telemetry/)*
+* [Peer Authentication](/docs/reference/config/security/peer_authentication/)
+
+## Telemetry API
+
+The `v1` Telemetry API is the only API that was promoted that had changes from its previous version. The following features weren’t promoted to `v1`:
```

Comment:
Worth clarify if below are the changes from previous version thus not being promoted to v1.

Reply:
@linsun does this help? 

_The `v1` Telemetry API is the only API that was promoted that had changes from its previous version. The following `v1alpha1` features weren’t promoted to `v1`_

Reply:
Thanks, yep that is better. Another clarification - previous version meaning previous API version, not Istio version?

Reply:
Correct, previous API version! I have updated it.